### PR TITLE
modify keyvault certificates starter readme

### PR DESCRIPTION
**Pull request by yiliuTo**
Modify README of `azure-spring-boot-starter-keyvault-certificates` to follow the specification of sdk repo.

### CODE REVIEW

**Comment by chenrujun** [comment]
After this PR merged, maybe we should create another PR to change `tenantId` to `tenant-id`.

Refs: https://github.com/Azure/azure-sdk-for-java/pull/17741

File: sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md [hunk 1]
```diff
@@ -262,17 +256,37 @@ To build and install the starter use the following command line:
...
+```
 
-# Key concepts
+For more information about setting logging in spring, please refer to the [official doc](https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-logging).
 
-# Examples
+## Next steps
+The following section provide a sample project illustrating how to use the starter.
+### More sample code
+- [Key Vault Certificates][sample]
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md:279]: Move `### More sample code` to `## Examples`.

File: sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md [hunk 2]
```diff
@@ -262,17 +256,37 @@ To build and install the starter use the following command line:
...
+## Contributing
+This project welcomes contributions and suggestions.  Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
 
-# Next steps
+Please follow [instructions here](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/CONTRIBUTING.md) to build from source or contribute.
 
-# Contributing
\ No newline at end of file
+<!-- LINKS -->
+[refdocs]: https://azure.github.io/azure-sdk-for-java/springboot.html#azure-spring-boot
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md:287]: Change all links to this style.

**Review by chenrujun — CHANGES_REQUESTED** [review]

File: sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md (hunk 1 above)
Comment (yiliuTo) [reply sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md:279]: The subsection here that displays a sample link follows sdk readme specifications, please refer to all other Spring READMEs.

**Review by chenrujun — APPROVED** [review]
LGTM.

**Pull request merged**